tests/functional/firefox/do_not_track.py

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from ...pages.firefox.do_not_track import DoNotTrackPage
+
+
+@pytest.mark.nondestructive
+def test_family_navigation(base_url, selenium):
+    page = DoNotTrackPage(base_url, selenium).open()
+    page.family_navigation.open_menu()
+    assert page.family_navigation.is_menu_displayed
```

Comment:
Nit: I would have an `open_family_navigation_menu` on `FirefoxBasePage` that returned a `FamilyNavigationMenu` object. This menu would then only have an `is_displayed` property. In my opinion this would make for a slightly more readable test:

```python
page = DoNotTrackPage(base_url, selenium).open()
menu = page.open_family_navigation_menu()
assert menu.is_displayed
```